api/views.py

```diff
@@ -24,18 +24,6 @@ class MeetingViewSet(viewsets.ModelViewSet):
 
 
 
-class MeetingsList3(views.APIView):
```

Comment:
@chriswedgwood  I deleted MeetingsList3 because it isn't referenced by anything.